.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,62 @@
+name: Build CI Wheels
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [
+          macos-latest,
+          windows-latest,
+          ubuntu-18.04,  # has to be the oldest possible for manylinux
+        ]
+      fail-fast: false
+
+    steps:
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - uses: s-weigand/setup-conda@v1
+        if: matrix.os == 'windows-latest'
+        with:
+          update-conda: true
+          conda-channels: conda-forge
+          activate-conda: true
+          python-version: '3.9'
```

Comment:
I'm not sure this is relevant in yt-astro, this comes directly from yt, I'll try to simplify it